storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/scheduling/TestGenericResourceAwareStrategy.java

```diff
@@ -181,10 +185,15 @@ public void testGenericResourceAwareStrategySharedMemory() {
     }
 
     /**
-     * test if the scheduling logic for the GenericResourceAwareStrategy is correct.
+     * Test if the scheduling logic for the GenericResourceAwareStrategy is correct
+     * without setting acker executors.
```

Comment:
It would be easier to understand  if we replace  "acker executors" with TOPOLOGY_ACKER_EXECUTORS here.